README.md

```diff
@@ -12,6 +12,15 @@ are unit tested and should be working as long as the build status is passing.
 
 - You will need to install [anaconda](https://www.anaconda.com/distribution/#download-section) first. 
 
+
+### macOS requirements
+
+lstchain depends on `protozfits` to read LST raw data via `ctapipe_io_lst`.
+Currently, prebuilt binaries are only available on linux.
+On macOS, you will need `protobuf` and `zeromq` installed so that the
```

Comment:
shouldn't we then include that in the environment? or at least an option for macos

Reply:
I am working on making it work with `protobuf` and `zeromq` installed via conda and also on a conda-forge recipe, that should also work for mac. Should be done tomorrow I hope.

Reply:
thanks